script/sort.py

```diff
@@ -0,0 +1,7 @@
+import argparse
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('fruit', nargs='*')
+args = parser.parse_args()
+print("Sorted list:", sorted(args.fruit))
```

Comment:
Please write in exactly this format:
`apple, banana, orange`
not like:
`Sorted list: ['apple', 'banana', 'grape', 'orange']`

Reply:
In that way it is ok?

Reply:
This file will not work. Try to run it and then fix the problem.